lib/src/extensions.dart

```diff
@@ -31,13 +34,21 @@ extension DateTimeTimeExtension on DateTime {
   /// Subtracts the Duration from this DateTime returns the difference as a new DateTime object.
   DateTime operator -(Duration duration) => subtract(duration);
 
+  /// Returns only year, month and day
+  DateTime get date => DateTime(year, month, day);
+
+  /// Returns only the time
+  String get timeOfDay =>
```

Comment:
This should probably return a `Duration` instead.

Reply:
Tjank you.